src/main/java/racingcar/car/CreateCar.java

```diff
@@ -0,0 +1,61 @@
+package racingcar.car;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static camp.nextstep.edu.missionutils.Console.readLine;
+import static camp.nextstep.edu.missionutils.Randoms.pickNumberInRange;
+
+public class CreateCar {
+    private final int carCount = 3;
```

Comment:
상수로 사용하는 경우에는 값이 변하는 경우는 없을 것이기에, static을 붙여서 사용하는 게 좋아보여요 [final vs static final](https://zorba91.tistory.com/275)
```suggestion
    private static final int carCount = 3;
```

Reply:
넵! 보내주신 링크 확인해 보니, static을 안 붙이면 호출될 때마다 할당되는 걸 알게 됐습니다!